### PR TITLE
NEXT-00000 - Add missing module warn message

### PR DESCRIPTION
**Pull request by ghost**
### 1. Why is this change necessary?
- When the media folder overview is loaded and a module for a media folder is missing, previously an error occurs (now there is no warn message) because it does not currently check whether the module exists before accessing it

### 2. What does this change do, exactly?
* Changed `sw-media-folder-item/index` to check if the module exists, if not display a warn message for developers

### 3. Describe each step to reproduce the issue or behaviour.
- Open the media folder overview with a missing module -> Previously: Error because the script is trying to access the manifest prop of the missing module, Now: No warn message from the missing module it just gets ignored

### 4. Please link to the relevant issues (if any).
- /

### 5. Checklist

- [X] I have rebased my changes to remove merge conflicts
- [X] I have written tests and verified that they fail without my change
- [X] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [X] I have written or adjusted the documentation according to my changes
- [X] This change has comments for package types, values, functions, and non-obvious lines of code
- [X] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/app/asyncComponent/media/sw-media-folder-item/index.js [hunk 1]
```diff
@@ -107,6 +107,11 @@ export default {
             }
 
             const module = this.moduleFactory.getModuleByEntityName(defaultFolder.entity);
+
+            if (!module) {
+                return;
```
Comment (shyim) [inline src/Administration/Resources/app/administration/src/app/asyncComponent/media/sw-media-folder-item/index.js:114]: maybe a console.warn with a proper hint for the developer 🤔

Comment (ghost) [reply src/Administration/Resources/app/administration/src/app/asyncComponent/media/sw-media-folder-item/index.js:114]: yes, we can add this. Which wording do you imagine for the warn? (And do we only include it in development (watcher) mode or also in prod?)

Comment (ghost) [reply src/Administration/Resources/app/administration/src/app/asyncComponent/media/sw-media-folder-item/index.js:114]: @shyim I added the warn message

**Comment by ghost** [comment]
@shyim It looks like the initial problem from this PR was fixed with https://github.com/shopware/shopware/commit/d12d4423dd0e2916d4e347dd2f4bb206a21bff8d#diff-7f512493902f72ede1d14ddbd7e4501ab1fff8b40782e3775bf4e7cbcf193ca7
Should this still get merged to display the warn message for developers?

**Comment by seggewiss** [comment]
> @shyim It looks like the initial problem from this PR was fixed with [d12d442#diff-7f512493902f72ede1d14ddbd7e4501ab1fff8b40782e3775bf4e7cbcf193ca7](https://github.com/shopware/shopware/commit/d12d4423dd0e2916d4e347dd2f4bb206a21bff8d#diff-7f512493902f72ede1d14ddbd7e4501ab1fff8b40782e3775bf4e7cbcf193ca7) Should this still get merged to display the warn message for developers?

We can still merge this change. I'll take care.

**Comment by seggewiss** [comment]
Thank you for contributing 😊

**Pull request closed without merging**